make_queue/views/reservation/reservation.py

```diff
@@ -14,6 +15,26 @@
 from make_queue.util.time import timedelta_to_hours
 from news.models import TimePlace
 
+from django.core.mail import send_mail, EmailMessage
+from django.http import HttpResponse
+
+
+
+class SLA_request(TemplateView):
+    template_name = "make_queue/SLA.html"
+
+    def post(self, request):
+        self.contact_info = request.POST.get("email")
+        self.file = request.POST.get("stl_file")
+        self.content = request.POST.get("description")
```

Comment:
There's no need to use `self` here because the values are only used in this function. Use normal variables instead

Reply:
Also, wouldn't it be better to use a Django form here? Both for cleaner code and backend checks on e.g. if an actual email is given.